src/test/java/com/cdancy/bitbucket/rest/features/RepositoryApiLiveTest.java

```diff
@@ -119,6 +119,18 @@ public void testListPermissionByGroup() {
         assertThat(permissionsPage.values()).isEmpty();
     }
 
+    @Test(dependsOnMethods = "testGetRepository")
+    public void testCreatePermissionByGroupNonExistent() {
+        boolean success = api().createPermissionsByGroup(projectKey, repoKey, "REPO_WRITE", randomString());
```

Comment:
This test is good, because It try to create permission group with group don't existent.